### PR TITLE
add tktable recipe

### PR DESCRIPTION
**Pull request by fhoehle**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/tktable/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - {{ cdt('libxcb') }}                # [linux]
+    - {{ cdt('libxau') }}                # [linux]
+  host:
+    - tk
+  run:
+    - tk
+
+test:
+  commands:
+    - sudo yum install xorg-x11-server-Xvfb -y  && echo "if {[catch {package require -exact Tktable {{ version }}; exit 0}]} {exit 1}" | xvfb-run tclsh  # [linux]
```
Comment (xhochy) [inline recipes/tktable/meta.yaml]: Use https://conda-forge.org/docs/maintainer/knowledge_base.html#yum-requirements-txt for this

**Comment by fhoehle** [comment]
@conda-forge-admin please rerender

**Comment by xhochy** [comment]
> @conda-forge-admin please rerender

This has not effect on staged-recipes, only on feedstocks ;)

**Comment by fhoehle** [comment]
resolving: https://github.com/conda-forge/r-base-feedstock/issues/64

**Comment by fhoehle** [comment]
@xhochy ready for review 🎉

File: recipes/tktable/license.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
+		   * COPYRIGHT AND LICENSE TERMS *
```
Comment (isuruf) [inline recipes/tktable/license.txt]: You can delete this file since it is in the tarball

Comment (fhoehle) [reply recipes/tktable/license.txt]: @isuruf here the corresponding PR, please review, thanks! https://github.com/conda-forge/tktable-feedstock/pull/1

**Pull request merged**